### PR TITLE
SEOImages: User Story 1463880, Part 24

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Files #1463880

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw This PR is ready for review. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019031800010/Report/

There are seven image names identified by CATS as being too short. They should be handled at a later time as they are not part of the SEO image without hyphen work.

**Comment by damabe** [comment]
Updated based on other PR review feedback. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019032600053/Report/

File: docs/framework/wpf/advanced/cleartype-registry-settings.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ This topic provides an overview of the [!INCLUDE[TLA2#tla_winclient](../../../..
   
  For each display device name for a user, a `TextContrastLevel` DWORD value is defined. The following screenshot shows the Registry Editor setting for the text contrast level.  
   
- ![ClearType settings in the Registry Editor](./media/cleartyperegistry01.png "ClearTypeRegistry01")  
+ ![Screenshot that shows ClearType settings in the Registry Editor.](./media/cleartype-registry-settings/cleartype-settings-registry-editor.png)
```
Comment (mairaw) [inline docs/framework/wpf/advanced/cleartype-registry-settings.md]:
```suggestion
 ![ClearType settings in the Registry Editor.](./media/cleartype-registry-settings/cleartype-settings-registry-editor.png)  
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Looks good @damabe. A few suggestions.

**Comment by damabe** [comment]
@mairaw Updated the PR. New CATS: https://contentqacats.azurewebsites.net/CheckResults/2019032900052/Report/

**Review by mairaw — APPROVED** [review]
Thank you @damabe!

**Pull request merged**